static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
That is odd that it still works. Obviously {{ conflicts with Django hence the [[ for existing templates.